db/migrate/20220908181655_add_dates_to_transaction_sync_batch.rb

```diff
@@ -0,0 +1,6 @@
+class AddDatesToTransactionSyncBatch < ActiveRecord::Migration[6.1]
+  def change
+    add_column :solidus_taxjar_transaction_sync_batches, :start_date, :date
+    add_column :solidus_taxjar_transaction_sync_batches, :end_date, :date
```

Comment:
I was originally thinking about us possibly storing these filters in a `serialized` column (similar to how we do preferences), so we don't have to migrate data when we add new filters. What are your thoughts on that approach, since it's unlikely we'll need to query on this data.

Reply:
In the interest of wrapping this ticket up I'm going to leave this for now. If we do end up updating this feature again this is a good suggestion.